### PR TITLE
Refactor Commands

### PR DESCRIPTION
**Pull request by Enrique-C**
Refactor of Commands

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java [hunk 1]
```diff
@@ -18,15 +18,26 @@
  * @author Enrique Carrizales.
  */
 public class CommandImageConverter implements ICommandStrategy{
+
+    // Content command value
+    private String commandValue;
+
     final String command = "convert";
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: you can use this value from Enum class

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java [hunk 2]
```diff
@@ -18,15 +18,26 @@
...
 
+    /**
+     * It builds the class.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: improve this comment

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java [hunk 3]
```diff
@@ -18,15 +18,26 @@
...
+
     /**
      * Builds a convert command.
-     * @param value receives a param.
      * @return command convert.
      */
     @Override
-    public String command(String value) {
+    public String command() {
         return this.SPACE + command;
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: change command to **commandValue**

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: But I don't have Enum class

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageRotate.java [hunk 4]
```diff
@@ -18,18 +18,28 @@
  *
  * @author Enrique Carrizales
  */
-public class CommandImageRotate implements ICommandStrategy{
+public class CommandImageRotate implements ICommandStrategy {
+
+    // Content command value
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/strategy/CommandImageRotate.java]: add period at end

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java (hunk 3 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: CommandImageConverter implements ICommandStrategy, it has String command(); method

File: src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/strategy/CommandImageConverter.java]: I did now.

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**